### PR TITLE
Fakebutler behavior

### PR DESCRIPTION
**Pull request by kantenkugel**
I am seeing no reason not to merge this into master after thinking about it.

Don't mind all the merge commits from master... this will be a squash merge.

Its using a separate listener for its "fake" behavior and that listener is removed on ready event, if the current bot is the main butler, so this shouldn't change anything for main butler

### CODE REVIEW

**Comment by DV8FromTheWorld** [comment]
Jeeeeeez those Master branch merges.

**Comment by kantenkugel** [comment]
Yea... thats what happens if you don't really care about a branch at all and only treat it like a temp deployment branch (for my butler alone)

File: src/main/java/com/kantenkugel/discordbot/fakebutler/FakeButlerListener.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    @Override
+    public void onEvent(Event event)
+    {
+        if (event instanceof UserOnlineStatusUpdateEvent)
+        {
+            UserOnlineStatusUpdateEvent e = (UserOnlineStatusUpdateEvent) event;
+            Guild guild = e.getGuild();
+            User user = e.getUser();
+            if(user.getIdLong() == REAL_BUTLER_ID && guild.getIdLong() == JDA_SERVER_ID)
```
Comment (Almighty-Alpaca) [inline src/main/java/com/kantenkugel/discordbot/fakebutler/FakeButlerListener.java:50]: As I don't see any way to turn off the fake listener this oiece of code will probably end in an endless loop on real butler

Comment (Almighty-Alpaca) [reply src/main/java/com/kantenkugel/discordbot/fakebutler/FakeButlerListener.java:50]: nvm... just saw looked a few lines down ^^

**Review by Almighty-Alpaca — APPROVED** [review]

**Pull request merged**